lib/job_tomate/events/jira/issue_updated_status.rb

```diff
@@ -16,6 +17,12 @@ class IssueUpdatedStatus
         # @param user_name [String]
         def run(issue, changelog, user_name)
           Actions::JIRAUpdateIssueAssigneeAndRolesForStatus.run(issue, changelog, user_name)
+          Actions::SlackNotifyOnAbsentFeatureEnv.run(issue, changelog) if notify_on_feature_env?(issue, changelog)
+        end
+
+        def notify_on_feature_env?(issue, changelog)
```

Comment:
Move the condition inside the action, events should not have business logic (sorry, not clear enough ;) !